pkg/scheduler/extender_test.go

```diff
@@ -365,8 +371,8 @@ func TestSchedulerWithExtenders(t *testing.T) {
 					return
 				}
 
-				if !reflect.DeepEqual(result, test.expectedResult) {
-					t.Errorf("Expected: %+v, Saw: %+v", test.expectedResult, result)
+				if diff := cmp.Diff(result, test.expectedResult, cmpOpts...); diff != "" {
+					t.Errorf("Unexpected result: (-want, +got): %s", diff)
```

Comment:
You should do new line before passing diff for better readability:
```suggestion
					t.Errorf("Unexpected result: (-want, +got):\n%s", diff)
```

Reply:
done